config/samples/catalogd_operatorcatalog.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: catalogd.operatorframework.io/v1alpha1
```

Comment:
would it be better to add a new sample rather than edit the existing one? Or are we subbing out image src in favor of oci?